changeset:   754:f64c2d57df43
branch:      0.6-maintenance
tag:         tip
user:        Toshio Kuratomi <toshio@fedoraproject.org>
date:        Tue Feb 22 12:05:49 2011 -0800
summary:     Fix for easy_install running on python-3.2

diff -r 5bc94690ee6c -r f64c2d57df43 setuptools/command/easy_install.py
--- a/setuptools/command/easy_install.py	Sat Nov 06 15:01:33 2010 +0100
+++ b/setuptools/command/easy_install.py	Tue Feb 22 12:05:49 2011 -0800
@@ -204,6 +204,12 @@
                             'exec_prefix': exec_prefix,
                            }
 
+        try:
+            self.config_vars['abiflags'] = sys.abiflags
+        except AttributeError:
+            # Only python-3.2+ has sys.abiflags
+            self.config_vars['abiflags'] = ''
+
         if HAS_USER_SITE:
             self.config_vars['userbase'] = self.install_userbase
             self.config_vars['usersite'] = self.install_usersite

